### DISCUSSION TITLE
Add Document permissions

### DISCUSSION THREAD

**Discussion by aam1r** [outline/outline#1634] [Ideas]
Collection permissions are really nice. 

Are Document permissions on the roadmap?

An example of this would be a a user has access to a collection that everyone has access to but there are certain documents shown to specific people.

**Comment by tommoor** [comment]
> Are Document permissions on the roadmap?

It definitely makes sense to add document permissions, in particular it has been requested to make sharing a draft between users easier. There isn't a timeline for this work at the moment though.

**Comment by matbrgz** [comment]
We are considering create sub collections instead of permissions per document.

**Comment by aam1r** [comment]
I think sub-collections is a great idea and more scalable in terms of usability and maintainability than having document-specific permissions. I like it 👍
